<launch>

 <!--  <master auto="start"/> -->
 
 <!--- Run AMCL --> 
   <include file="$(find indoor_thesis)/launch/amcl.launch" />

   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
    <rosparam file="$(find indoor_thesis)/Config_TEB/costmap_common_params.yaml" command="load" ns="global_costmap" /> 
    <rosparam file="$(find indoor_thesis)/Config_TEB/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find indoor_thesis)/Config_TEB/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find indoor_thesis)/Config_TEB/global_costmap_params.yaml" command="load" /> 
    <rosparam file="$(find indoor_thesis)/Config_TEB/teb_local_planner_params.yaml" command="load" />

    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/>
    <remap from="cmd_vel" to="/jackal_velocity_controller/cmd_vel"/>
    
 </node>

</launch> 